hbw/app/javascript/packs/hbw/components/form/file_list.js.jsx

```diff
@@ -33,16 +33,25 @@ modulejs.define('HBWFormFileList', ['React'], (React) => {
       const hiddenValue = JSON.stringify(links.filter(link => !deletedFiles.includes(link.name)));
 
       return <div className={cssClass}>
-        <input name={params.name} value={hiddenValue} type="hidden"/>
-        <label className={labelCSS}>
-          <span>{` ${label}`}</span>
-          <ul>
-            {this.files(links)}
-          </ul>
-        </label>
+        <div className="form-group">
+          <input name={params.name} value={hiddenValue} type="hidden"/>
+          <label className={labelCSS}>
+            <span>{` ${label}`}</span>
+            {params.description?.placement === 'top' && this.renderDescription()}
```

Comment:
I think we shouldn't provide a `placement` option if we want to always put it on top.